doc/keepalive.md

```diff
@@ -16,11 +16,11 @@ The above two channel arguments should be sufficient for most users, but the fol
 * **GRPC_ARG_KEEPALIVE_PERMIT_WITHOUT_CALLS**
   * This channel argument if set to 1 (0 : false; 1 : true), allows keepalive pings to be sent even if there are no calls in flight. 
 * **GRPC_ARG_HTTP2_MAX_PINGS_WITHOUT_DATA**
-  * This channel argument controls the maximum number of pings that can be sent when there is no other data (data frame or header frame) to be sent. GRPC Core will not continue sending pings if we run over the limit. Setting it to 0 allows sending pings without sending data.
+  * This channel argument controls the maximum number of pings that can be sent when there is no data/header/window_update frame to be sent. GRPC Core will not continue sending pings if we run over the limit. Setting it to 0 allows sending pings without such a restriction.
```

Comment:
Nit: to be pedantic is this strictly limited to `data/header/window_update` frames? Or are other h2 frame types, like `RST_STREAM` or settings, also considered?

I.e., should `data/header/window_update` be changed to something like `http2 frames (data frames, header frames, window_update frames, etc.)` ?

Reply:
No we don't consider others. For example, RST_STREAM doesn't affect this